spec/controllers/course/assessment/submission/answer/programming/annotations_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
     let(:course) { create(:course, creator: user) }
     let(:assessment) { create(:assessment, :with_programming_question, course: course) }
     let(:submission) do
-      create(:course_assessment_submission, :submitted, assessment: assessment, user: user)
+      create(:submission, :submitted, assessment: assessment, user: user)
```

Comment:
I prefer using the full form, because it tells you which model is the one being used. I wanted to actually remove all the shorthand aliases.